### ISSUE TITLE
Windows: running a cc_test requires presence of `//tools`

### ISSUE THREAD

**Issue by laszlocsomor** [bazelbuild/bazel#2754]
### Description of the problem / feature request / question:

`cc_test` rules can only be run if there's a `//tools` directory.

### If possible, provide a minimal example to reproduce the problem:

```
mkdir -p /tmp/foo && cd /tmp/foo

touch WORKSPACE

echo "cc_test(name='x',srcs=['x.cc'])" > BUILD

cat >x.cc <<EOF
#include <stdio.h>
int main(int,char**) { printf("PASS\n"); return 0; }
EOF
```

Then:
```
$ bazel test //:x
...
INFO: Found 1 test target...
ERROR: Source forest creation failed: C:/users/laszlocsomor/appdata/local/temp/_bazel_laszlocsomor/4itnmce4/execroot/foo/tools (No such file or directory).
INFO: Elapsed time: 0.440s
ERROR: Couldn't start the build. Unable to run tests.

$ mkdir tools

$ bazel test //:x
...
INFO: Elapsed time: 0.463s, Critical Path: 0.01s
//:x                                                            (cached) PASSED in 0.3s
```

### Environment info
* Operating System: Windows 10
* Bazel version (output of `bazel info release`): 0.4.5

**Comment by snnn** [comment]
I can reproduce this bug on :

Build label: 0.5.1- (@non-git)
Build target: bazel-out/msys_x64-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Sun Jun 18 12:29:37 2017 (1497788977)
Build timestamp: 1497788977
Build timestamp as int: 1497788977

**Comment by meteorcloudy** [comment]
Duplicated issue, #2474 
Solved by https://github.com/bazelbuild/bazel/commit/6c07525462062d491816a66a94c3db3b2dbc8f98
And the fix is in 0.5.2 already.
